lib/android/src/main/java/com/airbnb/android/react/lottie/ReactLottieAnimationView.java

```diff
@@ -0,0 +1,33 @@
+package com.airbnb.android.react.lottie;
+
+import android.animation.Animator;
+import android.content.Context;
+
+import com.airbnb.lottie.LottieAnimationView;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.uimanager.events.RCTEventEmitter;
+
+public class ReactLottieAnimationView extends LottieAnimationView implements Animator.AnimatorListener {
```

Comment:
why is this class necessary? why can't `LottieAnimationView` implement `Animator.AnimatorListener`